kolibri/deployment/default/settings/base.py

```diff
@@ -173,7 +173,11 @@
 # https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes
 # http://helpsharepointvision.nevron.com/Culture_Table.html
 
-LANGUAGES = i18n.get_supported_languages(KOLIBRI_MODULE_PATH)
+# Kolibri format
+KOLIBRI_SUPPORTED_LANGUAGES = i18n.get_supported_language_info(KOLIBRI_MODULE_PATH)
```

Comment:
I think this is better defined in the `i18n` module, as far as possible, we avoid defining Kolibri specific variables in the Django settings file, because the assumption is that they could, in principle, be overwritten by a third party user.